mathesar/database/types.py

```diff
@@ -3,18 +3,17 @@
 of db.types.base.DatabaseType).
 """
 from enum import Enum
-from collections.abc import Set, Collection, Mapping
-from typing import Optional
+from collections.abc import Collection
 from db.types.base import (
     DatabaseType, PostgresType, MathesarCustomType
 )
 from db.types.hintsets import db_types_hinted
 
 
 class UIType(Enum):
-    id: str
-    display_name: str
-    db_types: Collection[DatabaseType]
+    id: str  # noqa: NT001
+    display_name: str  # noqa: NT001
+    db_types: Collection[DatabaseType]  # noqa: NT001
```

Comment:
For clarity. These mixins only support objects that have these attributes. I didn't see another way to state that concisely and clearly. Also, this being actual type annotations, instead of a comment, helps with IDE complaining that referencing these attributes in this class is a mistake.

I'm open to getting rid of it, but I think this is a useful exception to the no-type-hints rule.